examples/overlays/package.json

```diff
@@ -8,7 +8,7 @@
     "mapbox-gl": "0.43",
```

Comment:
`react-map-gl` lock `mapbox-gl` version. Adding it to package.json will potentially install multiple copies.

Reply:
removed mapbox-gl from all examples. Thanks.